templates/documentation/sdks/player/apivideo-android-player-analytics.md

```diff
@@ -94,49 +71,27 @@ Options instantiation is made with either mediaUrl or videoInfo.
 Once the module is instantiated, the following methods have to be called to monitor the playback
 events.
 
-## ApiVideoPlayerAnalytics API
+### ApiVideoPlayerAnalytics API
 
-#### Event time or current time
+- Event time or current time
 
 If you know the event timestamp, you can use it as the `eventTime` parameter. If you don't know the
 event timestamp, you can set the `currentTime` parameter with a scheduler.
 
-#### `play(eventTime: Float = currentTime): Future<void>`
+#### Methods
 
-> method to call when the video starts playing for the first time (in the case of a resume after
-> paused, use `resume()`)
+| Method | Description |
+|----------|----------|
+| `play(eventTime: Float = currentTime): Future<void>` | method to call when the video starts playing for the first time (in the case of a resume after paused, use `resume()`) |
+| `resume(eventTime: Float = currentTime): Future<void>` | method to call when the video playback is resumed after a pause |
+| `ready(eventTime: Float = currentTime): Future<void>` | method to call once the player is ready to play the media |
+| `end(eventTime: Float = currentTime): Future<void>` | method to call when the video is ended |
+| `seek(from: Float, to: Float): Future<void>` | method to call when a seek event occurs, the `from` and `to` parameters are mandatory and should contain the seek start & end times in seconds |
+| `pause(eventTime: Float = currentTime): Future<void>` | method to call when the video is paused |
+| `destroy(eventTime: Float = currentTime): Future<void>` | method to call when the video player is disposed (eg. when the use closes the navigation tab) |
+| `currentTime` | field to call each time the playback time changes (it should be called often, the accuracy of the collected data depends on it) if you don't know event time. |
 
-#### `resume(eventTime: Float = currentTime): Future<void>`
-
-> method to call when the video playback is resumed after a pause
-
-#### `ready(eventTime: Float = currentTime): Future<void>`
```

Comment:
why did we remove this?

Reply:
I only moved these to a table format, the content is still there.